ons_ras_common/ons_environment.py

```diff
@@ -304,4 +320,16 @@ def rabbit(self):
 
     @property
     def debug(self):
-        return self._debug
\ No newline at end of file
+        return self._debug
+
+    @property
+    def asyncio(self):
```

Comment:
Do we need to access every single "service" from this object? It seems we have a single hard-coded representation that imposes how a microservice looks, whereas there is considerable variability in how the services really look.